boards/m4a-wroom/include/board.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ * Copyright (C) 2022 Mesh4all <mesh4all.org>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_m4a-esp32-wroom
+ * @brief       Board specific definitions for generic ESP32-WROOM-32 boards
+ * @{
+ * This configuration must to be used for only m4a-boards that implement
+ * ESP32-WROOM-32 model, this model implement the pin layouts to connect
+ * some sensors as ds18b20 and soil moisture sensor.
+ *
+ * @file
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @author      Eduardo Azócar <eduardo@turpialdev.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include <stdint.h>
+
+/**
+ * @name    Button pin definitions
+ * @{
+ */
+
+/**
+ * @brief   Default button GPIO pin definition
+ *
+ * Generic ESP32 boards have a BOOT button connected to GPIO0, which can be
+ * used as button during normal operation. Since the GPIO0 pin is pulled up,
+ * the button signal is inverted, i.e., pressing the button will give a
+ * low signal.
+ */
+#define BTN0_PIN        GPIO0
+
+/**
+ * @brief   Default button GPIO mode definition
+ *
+ * Since the GPIO of the button is pulled up with an external resistor, the
+ * mode for the GPIO pin has to be GPIO_IN.
+ */
+#define BTN0_MODE       GPIO_IN
+
+/**
+ * @brief   Default interrupt flank definition for the button GPIO
+ */
+#ifndef BTN0_INT_FLANK
+#define BTN0_INT_FLANK  GPIO_FALLING
```

Comment:
missing linear documentation for example 
```
#ifndef BTN0_INT_FLANK                     /*!< describe the define */

Reply:
It's not necessary, I toke a group of elements until button pin definition

Reply:
are different things, `@name` plus `{...}` is the name of the group, @brief or the inline comment, describes the definition